lib/presentation/view/movie_list_view.dart

```diff
@@ -12,7 +12,7 @@ class MovieListView extends StatefulWidget {
 }
 
 class _MovieListViewState extends State<MovieListView> {
-  final MovieRepository movieRepository = MovieRepository();
+  final MovieRepositoryImp movieRepository = MovieRepositoryImp();
```

Comment:
This should be `final IMovieRepository movieRepository = MovieRepositoryImp();`. 
The presentation layer relies on interfaces from the domain layer! 
(I know we're initializing this to an implementation from the data layer, but it's just now, we'll update that in a later PR)

Reply:
you are right! done!